internal/cfgplugins/mpls.go

```diff
@@ -238,3 +238,102 @@ func RemoveStaticMplsLspPushLabel(t *testing.T, dut *ondatra.DUTDevice, lspName
+	if deviations.MplsStaticPseudowireOcUnsupported(dut) {
+		cli := ""
+		switch dut.Vendor() {
+		case ondatra.ARISTA:
+			if params.Subinterface != 0 {
+				params.IntfName = fmt.Sprintf("%s.%v", params.IntfName, params.Subinterface)
+			}
+			cli = fmt.Sprintf(`
+			mpls pseudowires
+   				static pseudowires
+      			pseudowire %s
+         		transport nexthop-group %s
+         		local label %s
+         		neighbor label %s
+         		control-word
+			patch panel
+				patch patch-1
+				   connector interface %s
+				   connector pseudowire mpls static %s`,
+				params.PseudowireName, params.NexthopGroupName, params.LocalLabel, params.RemoteLabel, params.IntfName, params.PseudowireName)
+			helpers.GnmiCLIConfig(t, dut, cli)
+		default:
+			t.Errorf("Deviation MplsStaticPseudowireOcUnsupported is not handled for the dut: %v", dut.Vendor())
+		}
+		return
+	} else {
+		d := &oc.Root{}
+		fptest.ConfigureDefaultNetworkInstance(t, dut)
+		connectionCfg := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut)).GetOrCreateConnectionPoint("1")
+		endpointCfg := connectionCfg.GetOrCreateEndpoint("endpoint-1")
+		localCfg := endpointCfg.GetOrCreateLocal()
+		localCfg.SetInterface(params.IntfName)
+		localCfg.SetSubinterface(params.Subinterface)
+
+		remoteCfg := endpointCfg.GetOrCreateRemote()
+		remoteCfg.SetVirtualCircuitIdentifier(1001)
+
+		gnmi.BatchReplace(batch, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).ConnectionPoint("1").Config(), connectionCfg)
+	}
+}
+
+func RemoveMplsStaticPseudowire(t *testing.T, batch *gnmi.SetBatch, dut *ondatra.DUTDevice, params MplsStaticPseudowire) {
+	if deviations.MplsStaticPseudowireOcUnsupported(dut) {
+		cli := ""
+		switch dut.Vendor() {
+		case ondatra.ARISTA:
+			if params.Subinterface != 0 {
+				params.IntfName = fmt.Sprintf("%s.%v", params.IntfName, params.Subinterface)
+			}
+			cli = fmt.Sprintf(`
+			patch panel
+				patch patch-1
+				   no connector interface %s`,
+				params.IntfName)
+			helpers.GnmiCLIConfig(t, dut, cli)
+		default:
+			t.Errorf("Deviation MplsStaticPseudowireOcUnsupported is not handled for the dut: %v", dut.Vendor())
+		}
+		return
+	}
+}
+
+type VlanClientEncapsulationParams struct {
+	IntfName         string
+	Subinterfaces    uint32
+	RemoveVlanConfig bool
+}
+
+func VlanClientEncapsulation(t *testing.T, batch *gnmi.SetBatch, dut *ondatra.DUTDevice, params VlanClientEncapsulationParams) {
+	if deviations.VlanClientEncapsulationOcUnsupported(dut) {
+		cli := ""
+		if !params.RemoveVlanConfig {
+			cli = fmt.Sprintf(`
+					interface %v.%v
+						encapsulation vlan
+      						client dot1q %v network client
+						`, params.IntfName, params.Subinterfaces, params.Subinterfaces)
+		} else {
+			cli = fmt.Sprintf(`
+				interface %v.%v
+					no encapsulation vlan
+					`, params.IntfName, params.Subinterfaces)
+		}
+		helpers.GnmiCLIConfig(t, dut, cli)
+	} else {
+		// OC is not available
```

Comment:
@balaji6 your team should raise a pull request to propose the OC data model needed to cover this functionality.  Please contact me offline for support.